### PR TITLE
UG documentation for the Scheduler

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
UG documentation for the Scheduler.

Added the agenda view appointment template.
Added the vertical swiping for month view.

### CODE REVIEW

File: MAUI/Scheduler/agenda-view.md [hunk 1]
```diff
@@ -333,3 +333,42 @@ this.Content = scheduler;
 {% endhighlight %}
 {% endtabs %}
 
+## Agenda appointment template 
+
+The Syncfusion .NET MAUI Scheduler allows users to customize the visual representation of agenda appointments by defining data templates, enhancing usability within the application. Users can customize the appointment text, icon, image, or view based on our needs based on appointment details.
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/agenda-view.md:338]: The Syncfusion .NET MAUI Scheduler allows users to customize the visual representation of agenda appointments by defining data templates, enhancing usability within the application. Users can customize the appointment text, icon, image, or view based on their needs and appointment details.

File: MAUI/Scheduler/agenda-view.md [hunk 2]
```diff
@@ -333,3 +333,42 @@ this.Content = scheduler;
...
+                                   Text="{Binding StringFormat='{0:MMMM yyyy}'}" />
+                        </Grid>
+                    </DataTemplate>
+                </Scheduler:SchedulerAgendaView.MonthHeaderTemplate>
+            </Scheduler:SchedulerAgendaView>
+        </Scheduler:SfScheduler.AgendaView>
+    </Scheduler:SfScheduler>
+</Grid>
+
+{% endhighlight %}
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/agenda-view.md:373]: add the note section for the bindingcontext

For reference like this
The BindingContext of the CellTemplate is the SchedulerMonthCellDetails.

File: MAUI/Scheduler/month-view.md [hunk 3]
```diff
@@ -322,6 +322,29 @@ public class MonthViewHeaderTemplateSelector : DataTemplateSelector
...
+
+{% endhighlight %}
+{% highlight C# hl_lines="3" %}
+
+schedule.MonthView.NavigationDirection = SchedulerMonthNavigationDirection.Vertical;
+
+{% endhighlight %}  
+{% endtabs %}
+
+![Vertical Swiping MonthView](images\month-view\Swipe.gif)
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/month-view.md:346]: width="325" height="600" use width and height

File: MAUI/Scheduler/month-view.md [hunk 4]
```diff
@@ -322,6 +322,29 @@ public class MonthViewHeaderTemplateSelector : DataTemplateSelector
 N>
 * When using data template selector, performance issues occur as the conversion template views take time within the framework.
 
+## Month Navigation direction
+
+MonthView of SfScheduler can be navigated in both horizontal and vertical direction. You can change the direction of navigation through `NavigationDirection` property of MonthViewSettings in SfSchedule, by default Month navigation direction is `Horizontal`. If the direction is  `Vertical` the `ViewHeader` will be sticky.
```
Comment (saiganesh-sakthivel) [inline MAUI/Scheduler/month-view.md:327]: The `MonthView` of `SfScheduler` can be navigated in both horizontal and vertical directions. You can change the direction of navigation through the `NavigationDirection` property of `MonthViewSettings` in `SfScheduler`. By default, the navigation direction is `Horizontal`.

File: MAUI/Scheduler/agenda-view.md (hunk 1 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/agenda-view.md:338]: updated the changes

File: MAUI/Scheduler/month-view.md (hunk 4 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/month-view.md:327]: Updated the changes

File: MAUI/Scheduler/month-view.md (hunk 3 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/month-view.md:346]: updated the changes

File: MAUI/Scheduler/agenda-view.md (hunk 2 above)
Comment (SriRadheshNagS) [reply MAUI/Scheduler/agenda-view.md:373]: Updated the notes

**Review by saiganesh-sakthivel — APPROVED** [review]
fine

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**